compose/ui/ui/api/desktop/ui.api

```diff
@@ -3202,11 +3220,16 @@ public final class androidx/compose/ui/platform/ViewConfiguration$DefaultImpls {
 }
 
 public abstract interface class androidx/compose/ui/platform/WindowInfo {
-	public abstract fun getContainerSize-YbymL2g ()J
-	public abstract fun getKeyboardModifiers-k7X9c1A ()I
+	public fun getContainerSize-YbymL2g ()J
```

Comment:
It is the correct ABI, as it has a default implementation. The dump was dumped when we had incorrect ABI